manifests/gcp_marketplace/guide.md

```diff
@@ -98,3 +99,6 @@ as `Service Account User`. The Google Service Account is [Compute Engine default
 - Please also add your account as `Project Viewer` via [IAM](https://console.cloud.google.com/iam-admin/iam).
 
 For simplicity but not good for security, adding as `Project Editor` also can work.
+
+### Pipeline steps got insufficient permission
+Make sure following the procedure in [credential setup](#gcp-service-account-credentials). IAM configuration might take up to 5 mins to propagate.
```

Comment:
“might take up to 5 mins to propagate"
as I replied in Jessie's bug bash summarize email, it's not the truth. The script has issue and I fixed it in https://github.com/kubeflow/pipelines/pull/2493

Reply:
and better paste a sample error message like other tips for easy search. users may search Google with the error messages.

Reply:
by the way, "Enable API" is the step which take up to 5 min. Not the IAM step.

Reply:
> "might take up to 5 mins to propagate"

Oh actually this is not regarding Jessie's issue. It's a phenomenon observed by @IronPan and I last week. I remember we specified the role of `storage.admin` through UI instead of script. But thanks for the fix!

Will apply the other two suggestions.